otj-kafka/src/main/java/com/opentable/kafka/partition/InstanceKafkaPartitioningStrategy.java

```diff
@@ -42,6 +42,7 @@
  * NOTE: Normally the right thing to do is to use automatic rebalancing (Kafka default), which is a proven
  * scalable strategy that elastically scales. Use this class ONLY if there is a reason.
  */
+@SuppressWarnings("PMD")
```

Comment:
Please only exclude the specific violations. This does a PMD .* which is almost never justified